### ISSUE TITLE
Everquest .wld files

### ISSUE THREAD

**Issue by cjab** [martinlindhe/eqformat_map#1]
Hi there Martin,

I just ran across `eqformat_map` and `eqformat_pfs` on crates.io. I've been working on a wld file loader for a bit now and figured I would release it there also. It's pretty basic at the moment but I thought there might be some chance you would find it useful!

https://crates.io/crates/eq_wld

**Comment by martinlindhe** [comment]
That is pretty wild! I have some beginnings of a wld loader but it's no where near usable. I'll have a look at yours instead and probably scrap my effort. Thanks for letting me know!

**Comment by martinlindhe** [comment]
Would you consider licensing your crate under another license such as MIT or BSD instead of GPL-3 ?
If not, I'm afraid I won't be able to make use of it.

**Comment by cjab** [comment]
Yep! No problem. I've been _very_ slowly working on the pieces necessary to build a client in Rust. I know, a huge undertaking, but it's my only excuse for writing Rust right now :P. I also have an s3d extractor. It's not as far along as yours. It just ignores crc checks for example. But I went ahead and put it up on GitHub: https://github.com/cjab/eq_archive . Just curious what are you working on?

**Comment by martinlindhe** [comment]
I too am interested in building a client in rust, but never imagine to be able to do it all by myself so I figured I'd work on some separate components and hopefully stuff can be stitched together along the way.
I am currently working on a EQUI (xml ui files) renderer. And I'm in contact with a guy who's done a bunch of eq client networking code in rust

**Comment by cjab** [comment]
Nice! My plan is to continue work on rendering wld files. Although, probably a bit less now that the holidays are over :cry: . We should keep in touch though and try to combine our efforts 😃. Yeah the network client definitely seems daunting, that's a good contact to have 😃.

**Comment by martinlindhe** [comment]
Sounds good! You can reach me at martin@ubique.se, or more easily Nullius#3285 on Discord
